js/src/css/beautifier.js

```diff
@@ -68,11 +69,53 @@ function Beautifier(source_text, options) {
     "@document": true
   };
   this.NON_SEMICOLON_NEWLINE_PROPERTY = [
+    "grid-template-areas",
```

Comment:
You can put this in another PR and get it merged separate from this change.

Reply:
@bitwiseman Done #2035